### PR TITLE
Update rei and emi

### LINKED ISSUE #287: Missing items in REI
### What happened?

As the screenshot shows, I tried to search for Baby Yeti (I even have one), but simply no result. and recipe book works well. The same happens with Cropie,Wither cloak sword and others.
I have no idea why this happens sorry.

### Screenshots

sshot-2023-09-09-22-18-57

### Log output

[latest.log](https://github.com/SkyblockerMod/Skyblocker/files/12566062/latest.log)

### Minecraft Version

Minecraft 1.20.1

### Skyblocker Version

skyblocker-1.13.0+1.20.1.jar

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by LifeIsAParadox**
- [x] Added missing items to rei and emi fixes #287 
- [x] some texture are missing (example lighter blue Abicase)
Bildschirmfoto vom 2023-09-10 21-15-02

### CODE REVIEW

**Review by kevinthegreat1 — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -78,6 +79,9 @@ dependencies {
 
 	// Occlusion Culling (https://github.com/LogisticsCraft/OcclusionCulling)
 	include implementation("com.logisticscraft:occlusionculling:${project.occlusionculling_version}")
+
+	// neu repoparser
+	include implementation("moe.nea:neurepoparser:${project.repoparser_version}")
```
Comment (AzureAaron) [inline build.gradle:84]: Why are we importing and JiJing dependencies that we don't even use?

**Review by AzureAaron — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (kevinthegreat1) [reply build.gradle:84]: I think the goal is to start using this library? But you're right, we can include it later when we actually use it.

Comment (LifeIsAParadox) [reply build.gradle:84]: you are right i will remove it for now

Comment (LifeIsAParadox) [reply build.gradle]: i have commented it out

**Review by kevinthegreat1 — CHANGES_REQUESTED** [review]

**Review by kevinthegreat1 — APPROVED** [review]

**Review by AzureAaron — APPROVED** [review]

**Pull request merged**